requirements.txt

```diff
@@ -1,11 +1,11 @@
-Pyro4==4.31
-Twisted==19.7.0
-Whoosh==2.6.0
-argparse==1.2.1
-ijson==2.0
-psutil==2.1.3
-pycryptopp==0.6.0.1206569328141510525648634803928199668821045408958
-serpent==1.7
-defusedxml==0.4.1
-wsgiref==0.1.2
-zope.interface==4.1.2
+Pyro4
+Twisted
+Whoosh
+argparse
+ijson
+psutil
+pycryptopp
```

Comment:
There is no python3 version of that package, I would suggest to remove that dependency.

Reply:
I've created #248 to tackle that. There's code that uses this library that also needs to be removed.